packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -470,6 +478,49 @@ async function parseRequestObject(requestObject: IDataObject) {
 	return axiosConfig;
 }
 
+function digestAuthAxiosConfig(
+	axiosConfig: AxiosRequestConfig,
+	response: AxiosResponse,
+	auth: AxiosRequestConfig['auth'],
+): AxiosRequestConfig {
+	const authDetails = response.headers['www-authenticate']
+		.split(',')
+		.map((v: string) => v.split('='));
+	if (authDetails) {
+		const nonceCount = `000000001`.slice(-8);
```

Comment:
Why do we have a slice here? As it is a constant can we not just directly set it to the final value?

Reply:
For digest auth, if we have to reuse the nonce received in 401 api call then we can use dynamic nonceCount
But in our case as we always reinitialise the nonce for every api request, therefore static worked.
I was just confirming the nonceCount functionality and by mistake commited it to the branch, will remove it